components/RichTextEditor.js

```diff
@@ -245,6 +245,15 @@ export default class RichTextEditor extends React.Component {
       this.editorRef.current.addEventListener('trix-attachment-add', this.handleUpload);
       this.editorRef.current.addEventListener('trix-file-accept', this.handleFileAccept);
 
+      // We must listen when the user presses the 'Enter' key and when the user clicks the 'Link' button as well
+      const _this = this;
+      document.querySelector("[data-trix-input][name='href']").addEventListener('keydown', e => {
```

Comment:
Could we rather use refs? We have `this.editorRef` set above

Reply:
Initially I tried to use refs but I realized that **Trix** doesn't have an observer for the link button so I can't listen to that button from `this.editorRef`.

This is the Trix observer list: https://github.com/basecamp/trix#observing-editor-changes

Reply:
The problem with the current version is that if you have two editors on the same page, there's no guarantee that this code will target the correct one. We should have something like:

```suggestion
      this.editorRef.current.querySelector("[data-trix-input][name='href']").addEventListener('keydown', e => {
```